includes/definitions/svos.yaml

```diff
@@ -0,0 +1,9 @@
+os: svos
+text: 'Hitachi Storage Virtualization Operating System (SVOS)'
+type: storage
+icon: hds
+mib_dir:
+    - hitachi
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.116.3.11.4.1.1
```

Comment:
Sorry one last thing, this is quite specific sysObjectId. We don't have anything else registered in `1.3.6.1.4.1.116.` so you can probably use that, or be safe and do `1.3.6.1.4.1.116.3.` for now.